source/includes/administration/_pricings.md

```diff
@@ -1,6 +1,6 @@
 ## Pricings
 
-The pricing determines the price for each product defined for a selected catalog.
+The pricing determines the price for each product defined for a selected catalog. When prices are created, updated, or deleted with an effective date equal to or less than the current date a rollback is propagated to regenerate the affected prices.
```

Comment:
I think it would be more appropriate if we added this to the pricing change api doc. Do we have any documentation for this?

Reply:
There's an applied pricing section, I'll move the documentation over there because it's more applicable

Reply:
@Vioarr is it about the pricing changes api though? `/pricings/{pricingId}/changes` ?

Reply:
If there is no pricing changes doc we will need to create a task to write documentation around the changes api

Reply:
but then why is applied pricings talking about rollbacks?

Reply:
I just checked. That is out dated doc. 

I have created two new tasks
1. https://cloudops.atlassian.net/browse/MC-17405
2. https://cloudops.atlassian.net/browse/MC-17406

I think you should close this PR. I will post in redteam as well.

Reply:
closing, let's get to it!